File: EmployeeManagement.Domain/Models/Employee.cs
namespace EmployeeManagement.Domain.Models
{
    public class Employee
    {
        public int EmployeeId { get; set; }
        public string FirstName { get; set; } = "";
        public string LastName { get; set; } = "";
        public string Email { get; set; } = "";
        public string? PhoneNumber { get; set; }
        public DateTime HireDate { get; set; }
        public decimal Salary { get; set; }
        public int DepartmentId { get; set; }
        public string DepartmentName { get; set; } = string.Empty; 
        public bool IsActive { get; set; }
        public DateTime CreatedDate { get; set; }
        public int CreatedBy { get; set; }
        public int? UpdatedBy { get; set; }
        public DateTime? UpdatedDate { get; set; }
    }
}

File: Frontend/src/app/shared/models/employee.ts
export interface Employee {
  employeeId: number;
  firstName: string;
  lastName: string;
  email: string;
  phoneNumber?: string;
  hireDate: string;
  salary: number;
  departmentId: number;
  departmentName?: string;
  isActive?: boolean;
  createdDate?: string;
  createdBy?: number;
  updatedBy?: number;
  updatedDate?: string;
}